source/common/ipc/api.js

```diff
@@ -117,7 +117,7 @@ export type SubmitBugReportRequestMainResponse = void;
  * Channel to rebuild the electron application menu after the language setting changes
  */
 export const REBUILD_APP_MENU_CHANNEL = 'REBUILD_APP_MENU_CHANNEL';
-export type RebuildAppMenuRendererRequest = void;
+export type RebuildAppMenuRendererRequest = ?boolean;
```

Comment:
@tomothespian please change the type to: 
`export type RebuildAppMenuRendererRequest = { isAppUpdateAvailable: boolean };`